deepinv/optim/bregman.py

```diff
@@ -0,0 +1,198 @@
+import torch
+import torch.nn as nn
+from deepinv.optim.potential import Potential
+
+
+class Bregman(Potential):
```

Comment:
what is the difference between Bregman and Distance?

Reply:
Bregman forward calculates the Bregman Potential and not the Bregman divergence.